docs/setup/deploymentnew/v3-installation/1.2.1.0/aws-deployment.md

```
@@ -0,0 +1,462 @@
+
+* Wireguard Bastion server provides secure private channel to access MOSIP cluster.
+* Bastian server restricts public access, and enables access to only those clients who have their public key listed with it.
+* Bastian server listens on UDP port 51820.
+
+**Setup Wirguard VM**
+
+* Create a Wireguard server VM from aws console with mentioned '[**Hardware and Network Requirements**](pre-requisites.md)'.
+* Edit the security group and add the following inbound rules in aws console
+  * type ‘custom TCP', port range ‘51820’ and source '0.0.0.0/0’
+  * type ‘custom UDP', port range ‘51820’ and source '0.0.0.0/0’
+* Install docker in the Wireguard machine as given [here](https://docs.docker.com/engine/install/ubuntu/).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)